pkg/sql/types/types.proto

```diff
@@ -509,8 +524,6 @@ message InternalType {
     // GeoMetadata is populated for geospatial types.
     optional GeoMetadata geo_metadata = 14;
 
-    // StableTypeID is the ID of the backing TypeDescriptor for this type.
-    // StableTypeID is set only for User Defined Types.
-    optional uint32 stable_type_id = 15
-      [(gogoproto.nullable) = false, (gogoproto.customname) = "StableTypeID"];
+    // UDTMetadata is populated for user defined types.
+    optional SerializedUserDefinedTypeMetadata udt_metadata = 15 [(gogoproto.customname) = "UDTMetadata"];
```

Comment:
This is safe to change only because stable_type_id was added in in this release.